### PR TITLE
Application stack convert to speos

### PR DESCRIPTION
**Pull request by mcheng2022**

### CODE REVIEW

File: ansys_optical_automation/application/stack_convert_to_speos.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            file_path += extension.strip("*")
+    return file_path
+
+
+def main():
+    stack_test_version = 222
+    stack_result_file = getfilename("*.ldf")
+    if not os.path.exists(stack_result_file):
+        msg = "Cannot find specified file: " + stack_result_file
+        raise TypeError(msg)
```
Comment (pluAtAnsys) [inline ansys_optical_automation/application/stack_convert_to_speos.py:38]:
```suggestion
    if not stack_result_file:
        print("Exit programme, you have not select any files")
        return
```

File: ansys_optical_automation/application/stack_convert_to_speos.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from opcode import stack_effect
+import os
+import tkinter as tk
+from tkinter import filedialog
+
+from ansys_optical_automation.post_process.dpf_stack import DpfStack
+
```
Comment (pluAtAnsys) [inline ansys_optical_automation/application/stack_convert_to_speos.py:7]:
```suggestion


```

File: ansys_optical_automation/application/stack_convert_to_speos.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    stack_test_version = 222
+    stack_result_file = getfilename("*.ldf")
+    if not os.path.exists(stack_result_file):
+        msg = "Cannot find specified file: " + stack_result_file
+        raise TypeError(msg)
+    stack_dpf = DpfStack(stack_test_version)
+    stack_dpf.open_file(stack_result_file)
+    stack_dpf.convert_stack_to_speos()
+    print("Success: conversion of stack to Speos coating is done")
+
```
Comment (pluAtAnsys) [inline ansys_optical_automation/application/stack_convert_to_speos.py:43]:
```suggestion


```

Comment (pluAtAnsys) [reply ansys_optical_automation/application/stack_convert_to_speos.py:43]: As suggested by the failed pre-commmit style automation check, please run "pre-commit run -a" at terminal to make sure following Python PEP 8 standard

File: ansys_optical_automation/application/stack_convert_to_speos.py (hunk 2 above)
Comment (pluAtAnsys) [reply ansys_optical_automation/application/stack_convert_to_speos.py:7]: As suggested by the failed pre-commmit style automation check, please run "pre-commit run -a" at terminal to make sure following Python PEP 8 standard.

**Review by pluAtAnsys — CHANGES_REQUESTED** [review]

**Comment by pluAtAnsys** [comment]
@mcheng2022 I removed the unnecessary imports

**Comment by mcheng2022** [comment]
Thank you! That's weird I don't remember why the "from opcode..." line was added...

**Review by pluAtAnsys — APPROVED** [review]

**Pull request merged**